controller/users_blackbox_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	almtoken "github.com/almighty/almighty-core/token"
 
 	"context"
+
```

Comment:
I don't know if you read it bu there is a good practice regarding imports order. Generally they talk about having three groups:
1) golang pkgs
2) project pkgs
3) other github projects

WDYT ? I like that way, should we "enforce" it in our coide ?